### PR TITLE
Add Faker withdraw reasons

### PR DESCRIPTION
**Pull request by simonwhatley**
See: https://trello.com/c/6l38IW4Z/3062-show-feedback-for-withdrawn-applications-generated-with-faker

### CODE REVIEW

File: app/views/_components/offer-panel/template.njk [hunk 1]
```diff
@@ -5,7 +5,7 @@
   {% if params.withdraw or params.defer %}
     <p class="govuk-body">
       {% if params.withdraw %}
-        {# <a href="{{params.withdraw.href}}">Withdraw offer</a> #}
+        <a href="{{params.withdraw.href}}">Withdraw offer</a>
```
Comment (adamsilver) [inline app/views/_components/offer-panel/template.njk:8]: Lets leave this for a separate commit.

Comment (simonwhatley) [reply app/views/_components/offer-panel/template.njk:8]: I've commented it out from the offer page, so this won't be seen.
